### PR TITLE
Readium Web Navigator Implementation

### PR DESCRIPTION
**Pull request by bilal-uslu**
Readium Web Navigator Implementation

### CODE REVIEW

**Comment by bilal-uslu** [comment]
please note that, size-navigator / size (pull_request) is failed, which is normal, because we are going to add a pipeline at the first time

**Comment by chocolatkey** [comment]
What caused you to need to increase the size limit from 10KB to 50KB?

**Comment by bilal-uslu** [comment]
> What caused you to need to increase the size limit from 10KB to 50KB?

10 KB is initial value and it is 50 KB for shared already. The bundle size of this library is 14 KB now. If it goes over 50KB, it will give you a warning. Any size larger than 15 KB works at the moment. Here I choose 50 KB, which seems reasonable to me. This is to avoid unexpected packet sizes.

File: navigator/src/ui/Color.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  public toString() {
+    return `rgba(${[this.r, this.g, this.b, this.a].join(',')})`;
+  }
+
+  public static get black(): Color {
+    return new Color(0, 0, 0, 1);
+  }
+
+  public static get white(): Color {
+    return new Color(0, 0, 0, 0);
```
Comment (chocolatkey) [inline navigator/src/ui/Color.ts]: I think white should not be transparency, but RGBA all at max

Comment (bilal-uslu) [reply navigator/src/ui/Color.ts]: Right. Fixed.

**Pull request closed without merging**